app/services/romm_settings.py

```
@@ -0,0 +1,194 @@
+        _cache = merged
+        _token = stored_token if stored_token else None
+    return dict(merged)
+
+
+async def save(patch: dict[str, Any]) -> dict[str, Any]:
+    """Apply *patch* and persist it. Unknown keys are ignored.
+
+    Only the fields present in *patch* change, so a form that edits one card
+    cannot blank the settings another card owns.
+    """
+    global _cache, _token
```

Ruff (0.16.1)

[warning] Using the global statement to update `_cache` is discouraged

(PLW0603)